client/index.html.tpl

```diff
@@ -7,6 +7,11 @@
 
 	<link rel="preload" as="script" href="js/bundle.vendor.js">
 	<link rel="preload" as="script" href="js/bundle.js">
+	<!--we import the package manager first, so that it's cached for plugins and plugins can't override the object -->
```

Comment:
Why are they included in head and not after lounges code?

Reply:
It is after lounges code, it's after the bundle.

Reply:
This is preload. This should be at the bottom of the file

Reply:
Oh I missed that. Alrighty. I need to re-test this and make sure things actually work anyway, but yeah good point.